tools/grammar/src/test/resources/log4j.xml

```diff
@@ -1,19 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
-<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
-	<appender name="consoleAppender" class="org.apache.log4j.ConsoleAppender">
-		<layout class="org.apache.log4j.PatternLayout">
-			<param name="ConversionPattern" value="%d{HH:mm:ss.SSS} %-5p (%40F:%3L) - %m%n" />
-		</layout>
-	</appender>
-
-	
-	<logger name="org.opencypher.grammar.tools.antlr.BNFListener">
-		<level value="warn" />
-	</logger>
-	
-	<root>
-		<priority value="warn" />
-		<appender-ref ref="consoleAppender" />
-	</root>
-</log4j:configuration>
\ No newline at end of file
+<Configuration>

```

Comment:
I question whether this file is needed... I believe the `main/resources` are available to test executions as well, only that the `test/resources` can be used to override or provide additional resources.

Not that this is related to the change in this PR. I just happen to notice it since this PR made ma aware if the existence of this file.